### DISCUSSION TITLE
[PLUGIN REQUEST] Plugin to restart processes

### DISCUSSION THREAD

**Discussion by Crunchbits** [Flow-Launcher/Flow.Launcher#2157] [Plugin Request]
**I think it'd be good to either:**

1. Give the Process Killer plugin the option to restart a selected process via a shortcut like ctrl+enter or alt+enter or something else.

**OR**

2. Get a new plugin dedicated to restarting plugins so that you can use a specific action keyword like "restart".

I prefer the second one because it's potentially less destructive, unlike the first one where if you accidently miss the modifier+enter and now have a process that is killed and will not restart until you notice that it's not restarting and have to do it manually. Either way I think it'd be a good option.

**Comment by Garulf** [comment]
Sorry I'm not aware of any operating system that supports restarting generic processes. 

You may be able to restart a daemon/service.

**Comment by deefrawley** [comment]
Moving this to plugin request discussion board
